desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Ruffle
+GenericName=Flash Player
+Comment=Flash Player emulator
```

Comment:
Maybe use `An open source Flash Player emulator` as the comment (here and in the `metainfo.xml`)? That would be more consistent with the website and the comment should probably be nice to read.

Reply:
Unfortunately that would not conform to the [guidelines](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#summary):
* _The summary should ideally be between 10 and 25 characters, and no longer than 35 characters._
* _Don't duplicate metadata that's already available in a different form, such as the license, e.g. by mentioning that the app is "free and open source"._
* _Starting summaries with an article is almost always unnecessary. Instead, either omit the article or ideally phrase it in the imperative with a verb._

Reply:
Huh, interesting. I only looked at the official freedesktop specifications for the [desktop entry file](<https://specifications.freedesktop.org/desktop-entry-spec/latest/recognized-keys.html>) and [metainfo file](<https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html>) which are considerably less strict and have an example that's 43 characters.

But it's good that you noticed it :)

In any case, the desktop file specification states that the Comment name should not be redundant with the GenericName field, which is currently the case. Therefore I think it would still be good to improve it (and the description tag in `.metainfo.xml` somewhat.

Reply:
You're right! I missed that. Changed to `Play Flash content` as per our discussion on Discord